frontend/src/components/MainSearch/MainSearch.css

```diff
@@ -80,3 +80,43 @@
   font-weight: 500;
   margin: 0px;
 }
+
+.absoluteContainer {
+  /*display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100px;*/
```

Comment:
If we're not actually using this CSS, we probably don't need it. Is the div even necessary?